helper/random/string_generator.go

```diff
@@ -66,12 +66,17 @@ func sortCharset(chars string) string {
 	return string(r)
 }
 
-// StringGenerator generats random strings from the provided charset & adhering to a set of rules. The set of rules
+// StringGenerator generates random strings from the provided charset & adhering to a set of rules. The set of rules
 // are things like CharsetRule which requires a certain number of characters from a sub-charset.
 type StringGenerator struct {
 	// Length of the string to generate.
```

Comment:
I know we clarified it below, but it might be nice to state that this is the length of the _random_ portion of the password.